ister_gui.py

```diff
@@ -2253,7 +2310,7 @@ def _save_config(self, config, mount_d):
             if not mount_d[point]['format']:
                 if 'TYPE="' in subprocess.check_output(
                         'blkid | grep {0}'
-                        .format(disk+part), shell=True).decode('utf-8'):
+                        .format(disk+prefix+part), shell=True).decode('utf-8'):
```

Comment:
@bryteise this is the change that was needed. Prefix needs to hang around for this call, but partition needs to be a number or a string representation of a number by the time ister.py takes over.

Reply:
Ah okay, great.